ui/screens.py

```diff
@@ -1527,16 +1554,30 @@ def selectActivePool(self, pool):
             pool (VotingServiceProvider): The new active pool.
         """
         self.app.appWindow.showSuccess("new pool selected")
-        self.app.wallet.selectedAccount.setPool(pool)
+        self.app.wallet.selectedAccount.addPool(pool)
+        if not pool.isAccountless:
+            self.app.wallet.selectedAccount.setPool(pool)
         self.setPools()
 
     def addClicked(self, e=None):
         """
         The clicked slot for the add pool button. Stacks the pool screen.
         """
         self.app.appWindow.pop(self)
+        self.poolScreen.isAccountless = False
+        self.poolScreen.refreshAccountless()
+        self.app.appWindow.stack(self.poolScreen)
+
+    def addAccountlessClicked(self, e=None):
+        """
+        The clicked slot for the add pool button. Stacks the pool screen.
+        """
+        self.app.appWindow.pop(self)
+        self.poolScreen.isAccountless = True
+        self.poolScreen.refreshAccountless()
         self.app.appWindow.stack(self.poolScreen)
```

Comment:
these click events are tricky, wasn't sure how to pass a variable, so made a separate event for accountless, though seems wasteful